src/QMCHamiltonians/QMCHamiltonian.cpp

```diff
@@ -488,19 +488,19 @@ QMCHamiltonian::Return_t QMCHamiltonian::evaluate(ParticleSet& P)
   return LocalEnergy;
 }
 
-void QMCHamiltonian::flex_evaluate(const RefVector<QMCHamiltonian>& H_list,
-                                   const RefVector<ParticleSet>& P_list,
-                                   std::vector<Return_t>& LocalEnergies)
+std::vector<QMCHamiltonian::RealType> QMCHamiltonian::flex_evaluate(const RefVector<QMCHamiltonian>& H_list,
+                                   const RefVector<ParticleSet>& P_list)
 {
-  if (H_list.size() > 1)
-  {
+  // if (H_list.size() > 1)
```

Comment:
Could you restore this? This will be need for performance reason until we move H outside the crowd scope.

Reply:
I will when I add all the other things that need to be done so it doesn't break the driver.